examples/platforms/Makefile.am

```diff
@@ -55,6 +55,10 @@ if OPENTHREAD_EXAMPLES_DA15000
 SUBDIRS                                += da15000
 endif
 
+if OPENTHREAD_EXAMPLES_DA15000
```

Comment:
Intention of this commit was only to change README.md Something must go wrong with the process. I'll draw-back this pull request.